xAPI-About.md

```diff
@@ -493,29 +495,33 @@ that an LRS can take. It's also possible that some LRSs will have no mechanism f
 
-Finally, the set of Resources implemented is not expected to be constrained by this document. Resources beyond 
-those listed in this specification can be implemented and co-exist with the Resources defined in this specification.
+Finally, the set of resources (endpoints) implemented is not expected to be constrained by this document. Resources beyond 
+those listed in this specification can be implemented and co-exist with the resources defined in this specification.
 
 <a name="COPs" ></a>
 
 ## 7.0 Profiles, Vocabularies, and Communities of Practice
 
 xAPI strictly defines the structure of Statements, but is very flexible as to the contents of that structure. For example, the 
-specification requires that all Statements have a Verb property, but does not restrict the values of that property; any Verb 
+specification requires that all Statements have a "verb" property, but does not restrict the values of that property; any Verb 
 can be used. This flexibility enables xAPI to be used in any context, including future use cases not envisaged by the 
 specification authors.
 
-It is intended that [Communities of Practice (CoPs)](#def-community-of-practice) will define Verbs, Activity types, contextual relationships, Extensions, etc. 
-to be used in their profiles by providing unique identifiers wherever applicable. The CoP will define these identifiers and their metadata in a [Vocabulary](#def-vocabulary). A Profile is a set of rules and vocabularies to implemented in addition to xAPI for the particular use case being addressed. It is very important that such communities exist and share best practices. For more information on publishing vocabularies, see the [Vocabulary Companion Specification](https://adl.gitbooks.io/companion-specification-for-xapi-vocabularies/content) and [Vocabulary Primer](https://adl.gitbooks.io/experience-xapi-vocabulary-primer/content/) documents.
+It is intended that [Communities of Practice (CoPs)](#def-community-of-practice) will define Verbs, Activity types, 
+contextual relationships, extensions, etc. 
+to be used in their profiles by providing unique identifiers wherever applicable. The CoP will define these identifiers and 
+their metadata in a [Vocabulary](#def-vocabulary). A Profile is a set of rules and vocabularies to implemented in addition to 
+xAPI for the particular use case being addressed. It is very important that such communities exist and share best practices. 
+For more information on publishing vocabularies, see the [Vocabulary Companion Specification](https://adl.gitbooks.io/companion-specification-for-xapi-vocabularies/content) and [Vocabulary Primer](https://adl.gitbooks.io/experience-xapi-vocabulary-primer/content/) documents.
 
-It is recommended that a Profile use a unique "category" within a Statement's context to refer to any Statement  
+It is recommended that a Profile use a unique "category" within a Statement's "context" property to refer to any Statement  
```

Comment:
I prefer the original. By adding "property" it makes this part sound more technical than it is, perhaps giving the idea that category is a direct property of context.